Tests/LibWeb/Text/input/canvas/add-color-stop-color-function-should-not-crash.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<canvas id="canvas" width="100" height="50"> </canvas>
+<script>
+    const canvas = document.getElementById("canvas");
+    const ctx = canvas.getContext("2d");
+    var g = ctx.createLinearGradient(0, 0, 100, 0);
+    g.addColorStop(0, "color(srgb 1 0 0)");
+    g.addColorStop(1, "color(srgb 0 1 0)");
+    ctx.fillStyle = g;
+    ctx.fillRect(0, 0, 100, 50);
+</script>
+<script>
+    test(() => {
+        println("PASS (didn't crash)");
```

Comment:
You can put this in `Tests/LibWeb/Crash`, which is a separate category for crash tests. You can leave out this `println`.